angular-relative-date.js

```diff
@@ -60,7 +61,7 @@
         year = day * 365;
         delta = calculateDelta(now, date);
         if (delta > day && delta < week) {
-          date = new Date(date.getFullYear(), date.getMonth(), date.getDate(), 0, 0, 0);
+          date = new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate(), 0, 0, 0);
```

Comment:
I'm just trying to figure out the implications of this change. Won't this result in incorrect deltas for users outside of UTC? Perhaps I need to extend the test suite to verify this.